GUI/src/CMakeLists.txt

```diff
@@ -4,12 +4,18 @@ project(ElasticFusion)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}")
 
+if (WIN32)
+  find_package(JPEG REQUIRED)
```

Comment:
Surely this gets added as a requirement when you build Pangolin, and hence shouldn't be needed?